modules/tensor_mechanics/test/tests/shell/static/tests

```diff
@@ -19,7 +19,6 @@
     type = 'Exodiff'
     input = 'large_strain_m_40_AD.i'
     exodiff = 'large_strain_m_40_AD_out.e'
-
```

Comment:
The test is failing with this error:
```
tensor_mechanics/test:shell/static.large_beam_bending: *** ERROR ***
tensor_mechanics/test:shell/static.large_beam_bending: /opt/civet/build_0/moose/modules/tensor_mechanics/test/tests/shell/static/large_strain_m_40_AD.i:115: (Executioner/automatic_scaling):
tensor_mechanics/test:shell/static.large_beam_bending:     Automatic scaling requires a PETSc version of 3.9.0 or greater
```
To fix that, add this line for this test:
```
      petsc_version = '>=3.9.0'

Reply:
And the same for `large_beam_bending_jacobian`.